ssz/cache/utils.py

```diff
@@ -0,0 +1,60 @@
+import functools
+from typing import (
+    Any,
+    Iterable,
+)
+
+from eth_typing import (
+    Hash32,
+)
+from eth_utils import (
+    to_tuple,
+)
+
+from ssz.sedes.base import (
+    TSedes,
+)
+from ssz.typing import (
+    CacheObj,
+)
+
+
+def get_key(sedes: TSedes, value: Any) -> str:
+    key = _get_key(sedes, value).hex()
+    if len(key) > 0:
+        sedes_name = type(sedes).__name__
```

Comment:
I have the feeling that there should a better way to do this (e.g. define a `merkle_cache_key_prefix` class variable for all sedes objects, and maybe check that there's no name collision using a metaclass). But it's probably good enough for now.